### PR TITLE
Add new migration guide for atmos s3 to emc ecs s3

### PR DESCRIPTION
**Pull request by tstjep**

### CODE REVIEW

File: migrate-atmos-s3-to-ecs-s3.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,121 @@
+---
+title: Migrate from legacy ATMOS S3 to EMC ECS S3
+owner: Services
+---
+
+<strong><%= modified_date %></strong>
+
+This guide teaches you how to migrate from the soon-to-be-deprecated ATMOS S3 (dynstrg) to the new EMC ECS (dynstrg-2).
+
+In my space I have an instance of the old Atmos S3 Service (dynstrg):
```
Comment (annaronco) [inline migrate-atmos-s3-to-ecs-s3.html.md.erb]: Locate an instance of the old Atmos S3 Service (dynstrg) in your space:

File: migrate-atmos-s3-to-ecs-s3.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+<pre class="terminal">
+$ cf services
+Getting services in org INI-DOS-DPS-TCL_user / space playground as USER03...
+
+name                   service     plan    bound apps   last operation
+holiday-pictures       dynstrg     usage   my-awesome-app  create succeeded
+</pre>
+
+Checking the marketplace shows that there is a new S3 service:
```
Comment (annaronco) [inline migrate-atmos-s3-to-ecs-s3.html.md.erb]: Check the marketplace to see the new S3 service:

**Comment by Lafunamor** [comment]
I've added some clarifications, fixed some typos and updated the URLs

File: migrate-atmos-s3-to-ecs-s3.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+Check the marketplace to see the new S3 service:
+<pre class="terminal">
+$ cf marketplace | grep dynstrg
+dynstrg         usage*                       Dynamic Object Storage with S3 Interface based on ATMOS v2.2.3
+dynstrg-2       usage*                       Dynamic Object Storage with S3 API based on EMC ECS 3.0
+</pre>
+
+<strong>Let's migrate our Buckets to the new service:</strong>
+
+*Important*: Please make sure that you have the latest [cf CLI](https://github.com/cloudfoundry/cli) installed before you follow this migration guide.
```
Comment (annaronco) [inline migrate-atmos-s3-to-ecs-s3.html.md.erb]: Please, ...

File: migrate-atmos-s3-to-ecs-s3.html.md.erb [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+Creating service instance new-holiday-pictures in org INI-DOS-DPS-TCL / space playground
+</pre>
+
+Next, stop all your apps which use dynstrg so that no new objects are stored during the migration:
+
+<pre class="terminal">
+$ cf stop my-awesome-app
+</pre>
+
+As a next step, we'll have to download the minio S3 client. We are going to use it to copy one bucket after another recursively from the old service to the new one.
```
Comment (annaronco) [inline migrate-atmos-s3-to-ecs-s3.html.md.erb]: As a next step, you'll have to ... You will need it to copy ...

File: migrate-atmos-s3-to-ecs-s3.html.md.erb [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+For all operating systems, visit the minio website [minio](https://www.minio.io/downloads.html#download-client)
+
+To enable access to both services for our minio client, we get the credentials by generating a service key on each service:
```
Comment (annaronco) [inline migrate-atmos-s3-to-ecs-s3.html.md.erb]: ... your minio client, get the credentials ...

File: migrate-atmos-s3-to-ecs-s3.html.md.erb [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+<pre class="terminal">
+cf create-service-key holiday-pictures mys3servicekey
+cf create-service-key new-holiday-pictures mys3servicekey
+</pre>
+
+This will get us the credentials:
```
Comment (annaronco) [inline migrate-atmos-s3-to-ecs-s3.html.md.erb]: Run this command to get the credentials:

File: migrate-atmos-s3-to-ecs-s3.html.md.erb [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+                        "url": "https://ds11s3.swisscom.com",
+                        "accessKey": "85d9b397330add64920a1409f4732119-6683",
+                        "secretKey": "NXuYGhJnilomgZ9PpMbajTD85sn8s+twCBL5KvDq",
+                        "api": "S3v4",
+                        "lookup": "auto"
+                }
+}
+</pre>
+
+Before we can use the mc command (minio) to mirror a bucket, we need to create an empty bucket on the new service. In this example we have two buckets in our old service, "pictures" and "tickets":
```
Comment (annaronco) [inline migrate-atmos-s3-to-ecs-s3.html.md.erb]: Before you can use ... you need to ... In this example, you will have two buckets in your old service, "pictures" and "tickets":

File: migrate-atmos-s3-to-ecs-s3.html.md.erb [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+<pre class="terminal">
+mc mb holiday-dynstrg2/tickets
+Bucket created successfully `holiday-dynstrg2/tickets`.
+mc mb holiday-dynstrg2/pictures
+Bucket created successfully `holiday-dynstrg2/pictures`.
+</pre>
+ 
+Now we are finally able to migrate the whole buckets from the old to the new service:
```
Comment (annaronco) [inline migrate-atmos-s3-to-ecs-s3.html.md.erb]: Finally, you are able to migrate all the buckets from the old to the new service:

File: migrate-atmos-s3-to-ecs-s3.html.md.erb [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+
+<pre class="terminal">
+mc mirror holiday/tickets holiday-dynstrg2/tickets
+...bucket/ticket_japan.jpg:  18 B / 18 B
+</pre>
+
+To verify that everything succeeded list the new bucket:
```
Comment (annaronco) [inline migrate-atmos-s3-to-ecs-s3.html.md.erb]: To verify that everything succeeded, list the new bucket:

File: migrate-atmos-s3-to-ecs-s3.html.md.erb [hunk 10]
```diff
@@ -0,0 +1,121 @@
...
+
+As soon as everything is working properly, you can remove the service keys and the old service:
+
+<pre class="terminal">
+$ cf delete-service-key holiday-pictures mys3servicekey
+$ cf delete-service-key new-holiday-pictures mys3servicekey
+$ cf delete-service holiday-pictures 
+</pre>
+
+You now migrated your buckets to our new S3 service. Congrats! You can now bind your apps to the new service and start them again.
```
Comment (annaronco) [inline migrate-atmos-s3-to-ecs-s3.html.md.erb]: You now migrated your buckets to the new S3 service.

**Pull request merged**